L/LLVM/common.jl

```diff
@@ -786,6 +806,12 @@ function configure_extraction(ARGS, LLVM_full_version, name, libLLVM_version=not
             push!(products, ExecutableProduct("lld-link", :lld_link, "tools"))
             push!(products, ExecutableProduct("wasm-ld", :wasm_ld, "tools"))
         end
+    elseif name == "LLVM_utils"
+        script = llvm_utils_script
+        products = []
+        for tool in tools_list
+            push!(products, ExecutableProduct(tool, Symbol(tool), "tools"))
```

Comment:
Probably want to normalise `-` -> `_` in the Symbol?  Otherwise it becomes annoying to refer to (needs `var"..."` syntax)

Reply:
Also `count` apparently already exists so yeah.